### DISCUSSION TITLE
Conditional visibility of toggle fields

### DISCUSSION THREAD

**Discussion by dimdam** [filamentphp/filament#1301] [Ideas]
Is it possible to auto hide a toggle field if the specific form can only accept one value?
For example if we have a borrowing library app for a university, a new user can only be a teacher or a student.
If the administrator of this app toggles the "Is Teacher" field,then the "Is Student field" could deactivate. Or something like that.

**Comment by danharrin** [comment]
Yeah! On the student field, you can pass use `->hidden(fn ($get) => $get('is_teacher'))` or something.

https://filamentadmin.com/docs/2.x/forms/advanced#using-closure-customisation
